### DISCUSSION TITLE
Homework 8: Amount

### DISCUSSION THREAD

**Discussion by CS-MLU** [cafawo/FinancialDataAnalytics#18] [Q&A]
The method for selling one bitcoin introduced in the tutorial did not work reliably. There is on solution here: https://github.com/cafawo/FinancialDataAnalytics/discussions/17 but I run into the issue again in the second task which asks for 50% or 60% of the price of bitcoin. Does anybody have a reliable solution?

**Comment by birte0** [comment]
Please refer to the discussion in [#17](https://github.com/cafawo/FinancialDataAnalytics/discussions/17).  If it still doesn't work for you, please feel free to share your code here! :)
